code/conlleval.pl

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/perl -w
+# conlleval: evaluate result of processing CoNLL-2000 shared task
+# usage:     conlleval [-l] [-r] [-d delimiterTag] [-o oTag] < file
+#            README: http://www.clips.uantwerpen.be/conll2000/chunking/output.html
+# options:   l: generate LaTeX output for tables like in
+#               http://cnts.uia.ac.be/conll2003/ner/example.tex
```

Comment:
Maybe over-eager, but that URL does not work and seems to have been replaced by http://www.clips.uantwerpen.be/conll2003/ner/example.tex